src/main/resources/email/templates/digest.ftl

```diff
@@ -46,28 +46,29 @@
                         <#list categoryItem.itemList as item>
                             <#if item.dataSet?? && item.dataSet?size gt 0>
                                <div>
-                               <#list item.dataSet as item_key, item_value>
-                                   <#assign itemType="${item_key}">
                                    <div style="font-family: monospace;font-size: 14px;color: #445B68;padding-right: 15px;display: inline-block;">
-                                   <#if itemType == "RULE">
-                                     Rule: ${item_value}
-                                   <#elseif itemType == "COMPONENT">
-                                     Component: ${item_value} 
-                                   <#elseif itemType == "ADDED">
-                                     Added (${item_value})
-                                   <#elseif itemType == "UPDATED">
-                                     Updated (${item_value})
-                                   <#elseif itemType == "DELETED">
-                                     Deleted (${item_value})
-                                   <#elseif itemType == "PERSON">
-                                     By: ${item_value}
-                                   <#elseif itemType == "VERSION">
-                                     ${item_value}
-                                   <#else>
+                                   Component: ${item.dataSet.COMPONENT} 
```

Comment:
I added these changes to the email template, so the emails are always consistent in how they display component information